packages/inflection-tables/tests/data/greek-views.test.js

```diff
@@ -7,18 +7,18 @@ import { Constants, Feature, LanguageModelFactory } from 'alpheios-data-models'
 import BaseTestHelp from '@tests/data/base-test-help.js'
 
 describe('greek-views.test.js', () => {
-  console.error = function () {}
-  console.log = function () {}
-  console.warn = function () {}
+  //console.error = function () {}
```

Comment:
I think you could remove this comments signs

Reply:
good catch, thanks! will fix.